coalib/settings/ConfigurationGathering.py

```diff
@@ -187,8 +199,20 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
     log_printer.log_level = LOG_LEVEL.str_dict.get(str_log_level,
                                                    LOG_LEVEL.INFO)
 
-    warn_config_absent(sections, 'files', log_printer)
-    warn_config_absent(sections, 'bears', log_printer)
+    no_warning_options = (
+        'show_bears',
+        'show_capabilities')
+
+    try:
+        show_warnings = not any(
+            bool(cli_sections['default'].get(option, False))
+            for option in no_warning_options)
+    except ValueError:
+        show_warnings = False
```

Comment:
yeah I think we should log this, we shouldn't let this pass silently. (See also zen of python :)) If there's a non boolable setting that's supposed to be boolable the user should be informed